.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: php
 
+sudo: false
+
+cache:
+    - $HOME/.composer/cache/files
+
 install:
-    - travis_retry composer install --no-interaction --prefer-source
+    - travis_retry composer install
```

Comment:
this was done in https://github.com/travis-ci/travis-cookbooks/commit/ce17f7a2b61cf026e279ec6cc84913b79b908e71